File: src/main/java/top/zbeboy/isy/web/vo/graduate/design/reorder/DefenseOrderVo.java
package top.zbeboy.isy.web.vo.graduate.design.reorder;

import lombok.Data;

import javax.validation.constraints.Min;
import javax.validation.constraints.NotNull;
import javax.validation.constraints.Size;

/**
 * Created by zbeboy on 2017/7/25.
 */
@Data
public class DefenseOrderVo {
    @NotNull
    @Size(max = 64)
    private String defenseOrderId;
    @NotNull
    @Size(max = 64)
    private String defenseGroupId;
    @NotNull
    @Size(max = 64)
    private String graduationDesignReleaseId;
    @Min(0)
    private int defenseStatus;
}

File: src/main/java/top/zbeboy/isy/web/vo/graduate/design/reorder/DefenseOrderVo.kt
package top.zbeboy.isy.web.vo.graduate.design.reorder

import javax.validation.constraints.NotNull
import javax.validation.constraints.Size

/**
 * Created by zbeboy 2018-02-07 .
 **/
open class DefenseOrderVo {
    @NotNull
    @Size(max = 64)
    var defenseOrderId: String? = null
    @NotNull
    @Size(max = 64)
    var defenseGroupId: String? = null
    @NotNull
    @Size(max = 64)
    var graduationDesignReleaseId: String? = null
    var defenseStatus: Int = 0
    var grade: Double = 0.toDouble()
    var scoreTypeId: Int = 0
    var defenseQuestion: String? = null
}